.travis.yml

```diff
@@ -1,11 +1,46 @@
 # travis.yml for github.com/jts/nanopolish
 
-language: cpp
+dist: trusty
+sudo: false
```

Comment:
```
dist: trusty
sudo: false
```

These lines are for performance, to reduce the boot time.
See https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system